src/context.cpp

```diff
@@ -955,6 +1012,18 @@ namespace librealsense
         std::map< std::string, frame_metadata_syncer< rs2_software_motion_frame > > _stream_name_to_motion_syncer;
     };
 
+    // For cases when checking if this is< color_sensor > (like realsense-viewer::subdevice_model)
+    class dds_color_sensor_proxy : public dds_sensor_proxy, public color_sensor
```

Comment:
Check is we can add also `depth_sensor` 
Can be on the todo list

Reply:
Added